docs: add disclaimer for CTO approver (#546)
ansys/pyansys-dev-guide: doc/source/getting-started/administration.rst

        approval from the direct manager, product line, and the chief
        technology officer. No product source code is allowed.

.. note::

>    Chief technology officer approval is only needed for flagship-related libraries.

Author: Let me know if it works better now

>    BU chief technology officer approval is needed for multiphysics and tools impacting

Author: ```suggestion
   Business unit chief technology officer approval is needed for multiphysics and tools impacting
```

>    Business unit chief technology officer approval is needed for multiphysics and tools impacting

Author: ```suggestion
   Business unit chief technology officer approval is needed for multi-physics and tools impacting
```

> .. note::
>    Business unit chief technology officer approval is needed for multi-physics and tools impacting
>    several products. Flagship-related libraries require GM/VP approval.

Reviewer: Just suggesting to add a blank line here. Sometimes it causes issues:

```suggestion
.. note::

   Business unit chief technology officer approval is needed for multi-physics and tools impacting
   several products. Flagship-related libraries require GM/VP approval.
```

        approval from the direct manager, product line, and the chief
        technology officer. No product source code is allowed.

.. note::

>    Business unit chief technology officer approval is needed for multi-physics and tools impacting

Author: ```suggestion
   Chief technology officer approval is needed for multi-physics and tools impacting
```

>    Business unit chief technology officer approval is needed for multi-physics and tools impacting

Reviewer 2: I will say that having five adjectives before approval is really hard to digest! I'd suggest: For multi-physics and tools impacting several products, you must have approval from the chief technology officer for the Ansys business unit. For flagship-related projects, you must have general manager or vice president approval.

Is the "/" and "or" or an "and"--I went with "or" but may be incorrect. Also, my suggestion avoids using internal abrreviations, although I realize this topic is for Ansys employees. It's just best practice to avoid use of internal abbreviations in public-facing documentation.

Author: Sounds good, thanks @PipKat! Applying your suggestions right away. FYI - "or" was fine :)

        technology officer. No product source code is allowed.

.. note::

>    BU chief technology officer approval is needed for multiphysics and tools impacting

Reviewer 3: ```suggestion
   Chief technology officer approval is needed for multiphysics and tools impacting
```
